doc/kios.md

```diff
@@ -1 +1,871 @@
 # Kios Module
+Module | HTTP Method | URL | Description 
+--- | --- | --- | ---
+[Add](#add) | POST | /kios | Add Data Kios
+[Edit](#edit) | PUT | /kios/:id | Edit Data Kios
+[View](#view) | GET | /kios/:id | View Data Kios
+[Delete](#delete) | DELETE | /kios/:id | Delete Data Kios
+
+## <a name="add"></a>Add Data Kios
+
+### Endpoint
+POST /kios
+
+## Database
+![](./kios-layer.png)
+
+For add, you need get branch_code from branches tables
+
+### Header
+Key | Value 
+--- | ---
+Content-Type | application/json
+Accept | application/json
+
+### Request Payloads
+Name | Type | Example Value
+--- | --- | ---
+kios_name | string | mandala  
+address | string | jl. bekasi timur raya no. 159 A
+branch_code | string | mandala
+
+```
+{
+    "kios_name": "mandala",
+    "address": "jl. bekasi timur raya no. 159 A"
+    "branches": [
+        {
+            "branch_code": "02629"
+        }
+    ]
+}
+```
+
+### Response Payloads
+HTTP Code | Status | Description
+--- | --- | ---
+400 | Bad Request | Bad request payload  
+404 | Not Found | User not found in database  
+500 | Internal Server Error | some un-handle error in server 
+200 | OK | OK
```

Comment:
201 created

Reply:
i mean remove 200 and change with 201. for add will be created a new record, so, the success status code not just 200 ok, but specific status code : 201 created